pkg_resources.DistributionNotFound: The 'zest.releaser>=6.17.0' distribution was not found and is required by the application

I just tried to cut a release of plone-sphinx-theme with `fullrelease`. It got through most of the steps, but failed with the following traceback.

```
Traceback (most recent call last):
  File "/Users/stevepiercy/projects/plone-sphinx-theme/.venv/bin/fullrelease", line 10, in <module>
    sys.exit(main())
             ~~~~^^
  File "/Users/stevepiercy/projects/plone-sphinx-theme/.venv/lib/python3.13/site-packages/zest/releaser/fullrelease.py", line 23, in main
    prereleaser.run()
    ~~~~~~~~~~~~~~~^^
  File "/Users/stevepiercy/projects/plone-sphinx-theme/.venv/lib/python3.13/site-packages/zest/releaser/baserelease.py", line 426, in run
    self._run_hooks("before")
    ~~~~~~~~~~~~~~~^^^^^^^^^^
  File "/Users/stevepiercy/projects/plone-sphinx-theme/.venv/lib/python3.13/site-packages/zest/releaser/baserelease.py", line 423, in _run_hooks
    utils.run_hooks(self.zest_releaser_config, which_releaser, when, self.data)
    ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevepiercy/projects/plone-sphinx-theme/.venv/lib/python3.13/site-packages/zest/releaser/utils.py", line 594, in run_hooks
    run_entry_points(which_releaser, when, data)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevepiercy/projects/plone-sphinx-theme/.venv/lib/python3.13/site-packages/zest/releaser/utils.py", line 608, in run_entry_points
    plugin = entrypoint.load()
  File "/Users/stevepiercy/projects/plone-sphinx-theme/.venv/lib/python3.13/site-packages/pkg_resources/__init__.py", line 2775, in load
    self.require(*args, **kwargs)  # type: ignore[arg-type]
    ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/stevepiercy/projects/plone-sphinx-theme/.venv/lib/python3.13/site-packages/pkg_resources/__init__.py", line 2803, in require
    items = working_set.resolve(reqs, env, installer, extras=self.extras)
  File "/Users/stevepiercy/projects/plone-sphinx-theme/.venv/lib/python3.13/site-packages/pkg_resources/__init__.py", line 892, in resolve
    dist = self._resolve_dist(
        req, best, replace_conflicting, env, installer, required_by, to_activate
    )
  File "/Users/stevepiercy/projects/plone-sphinx-theme/.venv/lib/python3.13/site-packages/pkg_resources/__init__.py", line 933, in _resolve_dist
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'zest.releaser>=6.17.0' distribution was not found and is required by the application
make: *** [release] Error 1
```

Side note: I'm using Python 3.13, but the [trove classifiers show support for Python 3.8-3.11 only](https://github.com/zestsoftware/zest.releaser/blob/master/pyproject.toml#L32-L35). Should I create a separate issue to drop and add certain Python versions, and if so, what should be the supported Python version range?

--- a/zest/releaser/utils.py
+++ b/zest/releaser/utils.py
@@ -2,11 +2,11 @@
 
 from argparse import ArgumentParser
 from colorama import Fore
-from pkg_resources import parse_version
+from packaging import version
 
+import importlib
 import logging
 import os
-import pkg_resources
 import re
 import shlex
 import subprocess
@@ -184,7 +184,7 @@ def suggest_version(
             # too tricky.
             return
         if final:
-            parsed_version = parse_version(current)
+            parsed_version = version.parse(current)
             if not parsed_version.pre:
                 logger.warning(
                     "Version is not a pre version, so we cannot "
@@ -603,7 +603,7 @@ def run_entry_points(which_releaser, when, data):
 
     """
     group = f"zest.releaser.{which_releaser}.{when}"
-    for entrypoint in pkg_resources.iter_entry_points(group=group):
+    for entrypoint in importlib.metadata.entry_points(group=group):
         # Grab the function that is the actual plugin.
         plugin = entrypoint.load()
         # Feed the data dict to the plugin.
@@ -872,11 +872,11 @@ def get_last_tag(vcs, allow_missing=False):
     # Note: if parsing the current version fails, there is nothing we can do:
     # there is no sane way of knowing which version is smaller than an unparsable
     # version, so we just break hard.
-    parsed_version = parse_version(version)
+    parsed_version = version.parse(version)
     found = parsed_found = None
     for tag in available_tags:
         try:
-            parsed_tag = parse_version(tag)
+            parsed_tag = version.parse(tag)
         except Exception:
             # I don't want to import this specific exception,
             # because it sounds unstable:

